spec/requests/providers/means_summaries_spec.rb

```diff
@@ -82,84 +84,123 @@
   end
 
   describe "PATCH /providers/applications/:legal_aid_application_id/means_summary" do
-    subject do
+    subject(:request) do
       get providers_legal_aid_application_means_summary_path(legal_aid_application)
       patch providers_legal_aid_application_means_summary_path(legal_aid_application), params:
     end
 
     let(:params) { {} }
 
-    context "unauthenticated" do
-      before { subject }
+    context "when unauthenticated" do
+      before { request }
 
       it_behaves_like "a provider not authenticated"
     end
 
-    context "logged in as an authenticated provider" do
+    context "when logged in as an authenticated provider" do
       before do
         login
         allow(CFE::SubmissionManager).to receive(:call).with(legal_aid_application.id).and_return(cfe_result)
-        subject
       end
 
-      context "call to Check Financial Eligibility Service is successful" do
+      context "with a successful Check Financial Eligibility Service call" do
         let(:cfe_result) { true }
 
-        it "redirects along the successful path" do
-          expect(response).to redirect_to flow_forward_path
-        end
-
         it "transitions to provider_entering_merits state" do
+          request
           expect(legal_aid_application.reload.provider_entering_merits?).to be true
         end
 
-        context "when provider does not have bank_statement_upload permissions" do
+        context "when provider has bank_statement_upload permissions" do
```

Comment:
I guess you could update this context to explicitly say that enhanced bank upload is not enabled? minor nitpick though